glean-core/metrics.yaml

```diff
@@ -429,6 +429,22 @@ glean.error:
     no_lint:
       - COMMON_PREFIX
 
+  io:
+    type: counter
+    description: >
+      The number of times we encountered an IO error
+      when writing a pending ping to disk.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1686233
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1686233
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - glean-team@mozilla.com
+      - jrediger@mozilla.com
+    expires: never
```

Comment:
Another note: Because the code is not auto-generated we don't have any automatic way to expire those, so the only real thing is to keep them as never expiring.

Reply:
Oof. I wonder how much of this we'll end up mirroring for use counters in FOG.